README.md

```diff
@@ -20,11 +20,42 @@ Using the stock OSX version of git will likely cause problems. You probably want
     homesick symlink  dotfiles
 
 ### Optional
-If you're going to use hitch install it by running bundle in the dotfiles directory
+
+If you're going to use hitch or other ruby scripts in our dotfiles, install gem
+dependencies by running bundle in the dotfiles directory
```

Comment:
you're welcome to add that clarification to that line if you'd like, I was just adding the part about "other ruby scripts"